tests/netebpfext_unit/netebpf_ext_helper.cpp

```diff
@@ -76,6 +77,9 @@ _netebpf_ext_helper::_netebpf_ext_helper(
         nmr_hook_client_handle = std::make_unique<nmr_client_registration_t>(&hook_client, client_context);
         nmr_hook_client_handle_initialized = true;
     }
+
+    _fwp_engine::get()->set_sublayer_guids(
```

Comment:
nit: I think the `_fwp_engine::test_*` functions in `usersim/src/fwp_um.cpp` should be part of this repository.

Reply:
I will deal with that in the next PR in the series.  But offhand, I probably disagree.  Anything that allows a unit test to exercise WFP apis in a way that isn't ebpf specific would help any WFP client driver developer.

Reply:
Yes this is not blocking for this PR (I have already approved the PR).